### PR TITLE
Hh ham moblie looks moblie

### PR DESCRIPTION
**Pull request by hhassan1230**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,7 +13,8 @@
       <header>
         <div class="nav-bar">
           <div class="logo">
-            <img src="http://placehold.it/90x35">
+            <p>TJA | Tech Jobs Academy</p>
+            <!-- <img src="http://placehold.it/90x35"> -->
```
Comment (sarony) [inline index.html]: Let's remove this comment. We'll stick to just the text for now.

File: javascript/main.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ $("document").ready(function(){
     $('html, body').animate({
           scrollTop: $(clicked).offset().top - 35
       }, 1000);
+    return false;
```
Comment (sarony) [inline javascript/main.js:7]: Was it this `return false` that was causing the jump?

File: stylesheets/base.scss [hunk 3]
```diff
@@ -120,7 +86,7 @@ nav {
     margin-left: 10px;
     padding: 0px;
     counter-increment: my-awesome-counter;
-    margin-bottom: 10px;
+    // margin-bottom: 10px;
```
Comment (sarony) [inline stylesheets/base.scss]: Let's delete comments.

File: stylesheets/layout_desktop.scss [hunk 4]
```diff
@@ -45,4 +45,42 @@ section.curriculum .image-blurb {
 }
 .hero-blurb p {
   background-color: rgba(0, 0, 0, 0.03);
+}
+h3 {  font-size: modular-scale(0); }
+nav { ul { font-size: modular-scale(-1); } }
```
Comment (sarony) [inline stylesheets/layout_desktop.scss]: There's no reason to nest if we're just including one additional thing in the brackets, so let's remove the `{` after nav.

File: stylesheets/nav_1px.scss [hunk 5]
```diff
@@ -1,4 +1,13 @@
...
+    width: 100%;
+    text-align: center;
+    p { 
+      color: $green;
+      font-size: 8px;
+      margin-right: 0px;
+    }
+  }
   @include span-columns(12);
   @include clearfix;
```
Comment (sarony) [inline stylesheets/nav_1px.scss]: Let's move these `@include`s above under line 2

File: stylesheets/nav_1px.scss [hunk 6]
```diff
@@ -6,27 +15,31 @@
...
     float: left;
-    padding-left: 25px;
+
     display: inline;
-    a { color: white; }
+    a { 
+      color: white; 
+      padding-left: 5px;
+      padding-right: 5px;
+  }
```
Comment (sarony) [inline stylesheets/nav_1px.scss]: Looks like your indentation is off. Let's add another space here.

File: stylesheets/nav_desktop.scss [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+      margin-right: 0px;
+    }
+  }
+}
+nav {
+  float: right;
+  ul { 
+    font-family: $font;
+    font-size: 14px;
+    white-space:nowrap;
```
Comment (sarony) [inline stylesheets/nav_desktop.scss]: Didn't we decide this `white-space` was doing nothing? Let's remove it.

File: javascript/main.js (hunk 2 above)
Comment (hhassan1230) [inline javascript/main.js:7]: no return false fixes the jump problem.

**Pull request merged**